.travis.yml

```diff
@@ -17,6 +21,10 @@ addons:
     sources:
       - travis-ci/sqlite3
 
+jobs:
+  exclude:
+    - arch: ppc64le
+      python: 3.7
```

Comment:
why do we want to run it only on python 3.6? This version of python will be EOL soonish.

Reply:
For the focal distribution, the power support architecture currently does not support python 3.7 version. Power support architecture currently supports 3.5 and 3.6 in focal distribution. Thanks.